leftwm/src/bin/leftwm.rs

```diff
@@ -178,3 +189,15 @@ fn handle_help_or_version_flags(args: &[String], subcommands: &BTreeMap<&str, &s
     }
     app.get_matches_from(args);
 }
+
+fn send_lefthk_command(command: &str) {
+    let path = BaseDirectories::with_prefix("lefthk").expect("couldn't find base directory");
+    let pipe_file = path
+        .place_runtime_file("commands.pipe")
+        .expect("couldn't find commands.pipe");
+    let mut pipe = fs::OpenOptions::new()
+        .write(true)
+        .open(&pipe_file)
+        .expect("couldn't open file");
+    writeln!(pipe, "{}", command).expect("couldn't write to file");
+}
```

Comment:
Is this causing the reload issue? Why have you put this here and not in the `leftwm-worker`?

Reply:
As this is used to send kill to lefthk when leftwm-worker reloads.

Reply:
Ohh, `lefthk` not `leftwm`, should read more carefully...

But anyways, shouldn't follow this the same scheme as `leftwm` with `command-<$DISPLAY>.pipe`?

Reply:
Yep will update lefthk, may need to add a lefthk-command to handle it properly.

Reply:
This may need extra care as multiple lefthks will conflict slightly when using external commands.

Reply:
Oh, so the extra running `lefthk` instance might be the issue for blocking reloads as well right? I auto start this with xdg_autostart.

Reply:
I killed the standalone `lefthk` process and now reloading works again.